src/App.tsx

```diff
@@ -1,23 +1,36 @@
-import React from 'react';
-import logo from './logo.svg';
+import { useState, useEffect } from 'react';
+import { useQuery } from "@apollo/client";
 import './App.css';
+import { QUERY_USER } from './utils/graph_queries';
+import { User } from './utils/Models'
 
 function App() {
+  const { loading, error, data } = useQuery(QUERY_USER)
+  const [ user, setUser ] = useState<User | null>(null)
+
+  useEffect(() => {
+    if (!loading && data) {
+      setUser(data.fetchUser)
+    }
+  }, [data, loading])
+
+  if (loading) {
+   return  <h2>Loading</h2>
+  }
+
+  if (error) {
+    return <h2>{ error }</h2>
+  }
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+      <header className="header">
+        <h1>GlowUp</h1>
+        { user && <h2>Welcome { user.username }</h2>}
```

Comment:
This is here temporarily to prevent the warning of the user variable not being used.